ui/packages/catalog-ui-search/src/main/webapp/react-component/utils/save-file/save-file.tsx

```diff
@@ -14,12 +14,25 @@ const $ = require('jquery')
 export default function saveFile(name: string, type: string, data: any) {
   if (data != null && navigator.msSaveBlob)
     return navigator.msSaveBlob(new Blob([data], { type: type }), name)
-  var a = $("<a style='display: none;'/>")
-  var url = window.URL.createObjectURL(new Blob([data], { type: type }))
+  let a = $("<a style='display: none;'/>")
+  let url = window.URL.createObjectURL(new Blob([data], { type: type }))
   a.attr('href', url)
   a.attr('download', name)
   $('body').append(a)
   a[0].click()
   window.URL.revokeObjectURL(url)
   a.remove()
 }
+
+// return filename portion of content-disposition
+export function getFilenameFromContentDisposition(contentDisposition: any) {
```

Comment:
`contentDisposition` should be a `string` and the function's return type should be `string`.

Reply:
I can't set the function's return type to `string` because some cases return `null`.